pytorch_lightning/loops/optimization/optimizer_loop.py

```diff
@@ -144,12 +144,10 @@ def closure(self, *args: Any, **kwargs: Any) -> ClosureResult:
                 )
 
             if self._zero_grad_fn is not None:
-                with self._profiler.profile("zero_grad"):
```

Comment:
If we get rid of `profile("training_step_and_backward")` then we could remove the profiler reference from this class

Reply:
I think we could remove it. PyTorch Profiler would track those operations already at the PyTorch level.